test/cases/api/common/common.sh

```diff
@@ -38,6 +38,13 @@ function _instanceCheck() {
     set -eu
     [[ "$subscribe_org_id" == "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" ]]
 
+    FACTS=$($_ssh sudo subscription-manager facts)
+    if ! grep -q "image-builder.osbuild-composer.api-type: cloudapi-v2" <<< "$FACTS"; then
+        echo "$FACTS"
+        echo "System doesn't contain the expected osbuild facts"
```

Comment:
Let's reverse the order of these lines and do something like:

```bash
echo "System doesn't contain the expected `image-builder.osbuild-composer.api-type` fact or it has the wrong value."
sed -e 's/^/facts:/g' <<< "$FACTS"
```

Reply:
Do we actually gain anything from outputting all the facts if we didn't find ours? We already know it isn't in there.

Reply:
Not really I suppose, but maybe if one is misspelled slightly or something.

Reply:
how about `|grep image-builder` that way we only see ours?

Reply:
I like it.